modules/core/src/core/lib/layer.js

```diff
@@ -362,6 +362,71 @@ export default class Layer {
     }
   }
 
+  // This method overwrites the current buffer and is used for multi-picking
+  _clearInstancePickingColor(color) {
```

Comment:
These two methods are making a big assumption about how picking color is implemented. layers can overwrite the `instancePickingColor` updater however they want, or use an attribute that is named either `instancePickingColor` or `pickingColor`.

Reply:
My idea is that these layers should override:
- clearPickingColor()
- copyPickingColors()
- restorePickingColors()

And provide implementations that work with their internal logic.

Reply:
- If a method expects to be overridden, should we remove the underscore prefix?
- Looks like all we need is a method `getPickingColorAttribute` to generalize all of the above